status/flowcells.py

```diff
@@ -244,7 +255,13 @@ class FlowcellLinksDataHandler(SafeHandler):
     def get(self, flowcell):
         self.set_header("Content-type", "application/json")
         try:
-            p = lims.get_containers(name=flowcell[8:])[0]
```

Comment:
It's been done already

On Mon, Apr 27, 2015 at 12:49 PM, Francesco <notifications@github.com>
wrote:

> In status/flowcells.py
> <https://github.com/SciLifeLab/genomics-status/pull/197#discussion_r29136842>
> :
>
> > @@ -244,7 +255,13 @@ class FlowcellLinksDataHandler(SafeHandler):
> >      def get(self, flowcell):
> >          self.set_header("Content-type", "application/json")
> >          try:
> > -            p = lims.get_containers(name=flowcell[8:])[0]
>
> I understand your point... on the other hand a clear function name should
> avoid the need to look for what the function really does....
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SciLifeLab/genomics-status/pull/197/files#r29136842>.
>